Dockerfile

```diff
@@ -20,6 +20,13 @@ LABEL description="Dojo is a provable game engine and toolchain for building onc
     source="https://github.com/dojoengine/dojo" \
     documentation="https://book.dojoengine.org/"
 
+RUN apt-get -y update; \
```

Comment:
can you also move the `ENV HEALTH_CHECK_PORT` next to the `HEALTHCHECK` command? Generally speaking it's better to keep variable/declaration close to where they are used, and here it's not even in the final image if I'm right

Reply:
Gotcha, you're right, let me move it closer onto where it's used 👍